cmd/clusterawsadm/api/bootstrap/v1beta1/types.go

```diff
@@ -224,6 +231,10 @@ type AWSIAMConfigurationSpec struct {
 	// TODO: This field could be a pointer, but it seems it breaks setting default values?
 	// +optional
 	S3Buckets S3Buckets `json:"s3Buckets,omitempty"`
+
+	// IPv6 when enabled, will add IPv6 specific permissions.
+	// +optional
+	IPv6 *IPv6 `json:"ipv6,omitempty"`
```

Comment:
I'm wondering if we should enable IPv6 by default. If we do, then this turns into a way to disable the permissions instead of enabled them. wdyt?

Reply:
If it's not enabled by default, when do we change it to be the default in the future?

Reply:
Enabling ipv6 by default is a non-goal, FWIW. It's experimental at best, right? People will need to keep testing it as I'm sure, some things will not work. :)

Reply:
>when do we change it to be the default in the future?

Maybe half a year from now... ?

Reply:
If we think its experimental then we should also add a feature flag to fgo along with the opt-in permissions?

Reply:
Isn't the fact that you explicitly have to set `vpc: ipv6{}` a kind of feature flag already? ( I don't know, just genuinely asking :) ).